cpu/atmega_common/include/atmega_regs_common.h

```diff
@@ -1,23 +1,25 @@
 /*
  * Copyright (C) 2016 Freie Universität Berlin
+ *               2016 INRIA
```

Comment:
For consistency, one should write `Inria`. [Ref](http://www.inria.fr/en/content/view/full/11297), see the `Reminder section`. There are a few others.